src/shapes/text.class.ts

```diff
@@ -381,25 +391,8 @@ export class Text<
   setPathInfo() {
     const path = this.path;
     if (path) {
-      path.segmentsInfo = getPathSegmentsInfo(path.path);
-    }
-  }
-
-  /**
-   * Return a context for measurement of text string.
-   * if created it gets stored for reuse
-   * this is for internal use, please do not use it
-   * @private
-   * @param {String} text Text string
-   * @param {Object} [options] Options object
-   */
-  getMeasuringContext() {
-    if (!fabric._measuringContext) {
-      fabric._measuringContext =
-        (this.canvas && this.canvas.contextCache) ||
-        createCanvasElement().getContext('2d');
+      this.pathSegmentsInfo = getPathSegmentsInfo(path.path);
```

Comment:
i really prefer this to stay in path. It makes easier to swap paths and also the path can use those segment infos for other stuff in the future, since this is info of the path itself.

Reply:
legit
so we type it on path? or we type text with 
```ts
path: Path & { info: WhatEver }
```

Reply:
i typed in on path, if the build pass i ll merge this PR